concepticondata/conceptlists/Desrochers-2010-330.tsv

```diff
@@ -0,0 +1,331 @@
+Desrochers-2010-330-193	193	mundo	world	WORLD	965	Noun	6.44117647058824	5	3783	6.44
+Desrochers-2010-330-194	194	nieve	snow	SNOW	784	Noun	5.82352941176471	5	220	5.41
+Desrochers-2010-330-195	195	nuez	nut	NUT	191	Noun	5.42156862745098	4	24	5.17
+Desrochers-2010-330-196	196	ola	wave	WAVE	978	Noun	5.5	3	62	6.76
+Desrochers-2010-330-197	197	página	page			Noun	6.52941176470588	6	178	6.64
+Desrochers-2010-330-198	198	pájaro	bird	BIRD	937	Noun	6.29411764705882	6	116	6.5
+Desrochers-2010-330-199	199	papel	paper	PAPER	648	Noun	6.66666666666667	5	1028	6.66
+Desrochers-2010-330-200	200	paso	step	STEP	1158	Noun	5.88235294117647	4	1331	6.24
+Desrochers-2010-330-201	201	pata	paw	PAW	2965	Noun	5.92156862745098	4	78	6.45
+Desrochers-2010-330-202	202	patata	potato	POTATO	593	Noun	6.51960784313726	6	33	6.75
+Desrochers-2010-330-203	203	pelota	ball	BALL	1845	Noun	6.40196078431373	6	103	6.49
+Desrochers-2010-330-204	204	perra	bitch			Noun	6.22	5	60	6.47
+Desrochers-2010-330-205	205	persona	person	PERSON	683	Noun	6.50980392156863	7	962	7
+Desrochers-2010-330-206	206	pizza	pizza			Noun	6.24509803921569	5	5	
+Desrochers-2010-330-207	207	plato	plate	PLATE	480	Noun	6.6078431372549	5	173	
+Desrochers-2010-330-208	208	pluma	pen	PEN	794	Noun	5.48039215686275	5	89	5.76
+Desrochers-2010-330-209	209	puerta	door	DOOR	1567	Noun	6.57843137254902	6	1557	6.71
+Desrochers-2010-330-210	210	punta	tip	TIP (OF OBJECT)	2992	Noun	5.89114925588637	5	232	5.69
+Desrochers-2010-330-211	211	puro	pure			Noun	5.42156862745098	4	236	5.33
+Desrochers-2010-330-212	212	rabia	rabies	RABIES	2206	Noun	5.51960784313726	5	128	6.18
+Desrochers-2010-330-213	213	rata	rat	RAT	1490	Noun	5.79411764705882	4	70	6.3
+Desrochers-2010-330-214	214	rayo	ray			Noun	5.15686274509804	4	111	5.86
+Desrochers-2010-330-215	215	sello	stamp	POSTAGE STAMP	596	Noun	6.09915634874148	5	35	6.61
+Desrochers-2010-330-216	216	submarino	submarine			Noun	4.57843137254902	9	12	
+Desrochers-2010-330-217	217	techo	roof	ROOF	769	Noun	5.99019607843137	5	225	6.54
+Desrochers-2010-330-218	218	torpedo	torpedo			Noun	3.90196078431373	7	4	
+Desrochers-2010-330-219	219	uña	nail	FINGERNAIL	1258	Noun	6.23529411764706	3	24	6.27
+Desrochers-2010-330-220	220	uva	grape	GRAPE	920	Noun	5.80392156862745	3	25	
+Desrochers-2010-330-221	221	vajilla	tableware			Noun	5.63725490196079	7	17	
+Desrochers-2010-330-222	222	vampiro	vampire			Noun	4.17647058823529	7	22	4.25
+Desrochers-2010-330-223	223	vaso	glass	GLASS	604	Noun	6.58823529411765	4	209	6.3
+Desrochers-2010-330-224	224	vela	sail			Noun	5.67647058823529	4	83	5.78
+Desrochers-2010-330-225	225	vida	life	LIFE	210	Noun	6.59803921568627	4	4765	6.44
+Desrochers-2010-330-226	226	vino	wine	WINE	1524	Noun	6.35294117647059	4	716	6.39
+Desrochers-2010-330-227	227	abatir	download			Verb	3.77450980392157	6	5	
+Desrochers-2010-330-228	228	abrir	open			Verb	6.55882352941176	5	275	6.45
+Desrochers-2010-330-229	229	acudir	go to			Verb	4.82352941176471	6	141	4.91
```

Comment:
too concrete, it refers rather to "sich an jemanden wenden" or similar, so "to attend to somebody"

Reply:
I suppose the closest is indeed 1897 | HELP

Reply:
We mapped Spanish "ayudar" to [HELP](https://concepticon.clld.org/parameters/1897) already. I'd leave it unmapped if there is no fitting concept.

Reply:
Yes, can't find it anything closer. RESCUE is perhaps the closest, but not good enough (it would be "resgatar").